### ISSUE TITLE
Recently updated pyspedas causes issues with MMS loading functions

### ISSUE THREAD

**Issue by J-Beedle** [spedas/pyspedas#152]
I recently updated my version of pyspedas (last updated back in mid-2020) and I have encounter several issues with the MMS load routines. For example, if I try calling pyspedas.mms.fgm(), I receive several lines of feedback that some variables are not loaded into pytplot (as can be seen in the attached screenshot using Jupyter). I am not sure if this is impacting the data loaded into the variables that were able to make it into pytplot, but this is the first time I have seen an output like this. 

<img width="852" alt="Screen Shot 2022-05-20 at 4 20 41 PM" src="https://user-images.githubusercontent.com/105941980/169605114-e2f48763-3f65-48aa-bdc4-570f8bec1a7a.png">

Additionally, it seems that several previously provided MMS fgm variables are no longer being provided by the mms_fgm_load function. Most critically in my case, my code requires the use of the "mms#_fgm_r_gse_brst_l2" data files to calculate the curl of the magnetic field as measured by the MMS spacecraft. In the previous build of pyspedas, I was able to do this by calling mms_load_fgm(probe=probes,data_rate='brst',trange=trHr,varformat='*_r_'+'gse'); however, when I call this loading function now, I get a long list of "That name is currently not in pytplot" and no loaded variables. This is illustrated in the second screenshot which was outputted from Spyder using Python 3.9.

Is there a way to access the  "mms#_fgm_r_gse_brst_l2" using a different loading function in this current pyspedas build, or is there a way where I could revert my version of pyspedas to the previous build?

Thanks!

<img width="400" alt="Screen Shot 2022-05-20 at 4 54 59 PM" src="https://user-images.githubusercontent.com/105941980/169609503-9fd2eed3-a89c-434d-a5f9-256bd7a72b45.png">

**Comment by supervised** [comment]
Hello!

You can access these variables by setting the `get_fgm_ephemeris` keyword to `True`, e.g., 

`pyspedas.mms.fgm(get_fgm_ephemeris=True, varformat='*_r_gse*')`

One important note: these variables aren't necessarily monotonically increasing; if I recall correctly, these variables can have extra data points at the start/end of each CDF (so it should be monotonically increasing as long as you only load 1 CDF). You might want to use the MMS ephemeris data (pyspedas.mms.mec) for the official data. 

I've also removed the 'That name is currently not in pytplot' warnings for FGM (as of v1.3.7).

Hope this helps!

**Comment by J-Beedle** [comment]
Thank you for the help! Your advice regarding the ephemeris keyword worked like a charm, and thank you for the note regarding the fgm ephemeris data. I compared the current densities calculated from the FPI current method with the ones that I obtained from the curlometer method using this fgm ephemeris data, and it looks like the outputs are essentially identical over the timeframes that I am considering. But I will keep this in mind if I start working with longer timeframes.
